sdk/file_sdk/XMPFileFormat/xmp_file_format.h

```diff
@@ -1,6 +1,9 @@
 #ifndef XMP_FILE_FORMAT_H
 #define XMP_FILE_FORMAT_H
 
+#define TXMP_STRING_TYPE std::string
```

Comment:
can we use std::wstring?

Reply:
No

---
In reply to: [136425305](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136425305) [](ancestors = 136425305)

Reply:
How does Unicode works?

Reply:
wchar*? what can we use to support unicode?

---
In reply to: [136711974](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136711974) [](ancestors = 136711974,136425305)